migrate/migrate.py

```diff
@@ -73,9 +73,9 @@ def migrate(next_db_version, db_cur, db_conn):
 
 def update_version(current_db_version, db_cur, db_conn):
     print("Updating db version to {}...".format(current_db_version))
-    cmd = "UPDATE db_details SET version={0}, upgraded_on=now();".format(current_db_version)
+    cmd = "UPDATE db_details SET version=%s, upgraded_on=now();"
```

Comment:
is this properly formatted as an integer instead of a string?

Reply:
%s is a positional placeholder and it keeps the type of the variable